creator-node/src/routes/tracks.js

```diff
@@ -690,8 +691,8 @@ module.exports = function (app) {
             fileRecord = redisFileRecord
           }
         }
-      } catch (e) {
-        req.logger.error(`Error looking for stream fallback in redis`, e)
+      } catch (error) {
+        req.logger.error({err: error}, 'Error looking for stream fallback in redis')
```

Comment:
ahhh I meant `{error: e}`. the `error` key will be part of the serializer and I thought having the full name would be more clear